### PR TITLE
Add file download component

### LINKED ISSUE #315: Create File download component
Create File download component and add it to Design system on Netlify. 

All components are listed in Zeplin Styleguides https://app.zeplin.io/styleguide/5d95b0aaad77af16c796a476/components?coid=5f3a7b7fe11ed48072fe78ce

- [ ] Accessible file download
- [ ] Not accessible file download

### PR DESCRIPTION
**Pull request by gldgrnt**
Fixes #315

### CODE REVIEW

File: src/wmnds/components/file-download/_file-download.njk [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            icon: "general-file"
+        }) | trim | indent(4)
+    }}
+    <div class="wmnds-file-download__desc">
+        <a href="#" download="{{ fileName|replace(' ', '_') + '.' + fileType|lower }}" class="wmnds-link">
+            Download '{{fileName}}' ({{fileType}}, {{fileSize}})
+        </a>
+        {% if params.accessible === false -%}
+        <p class="wmnds-file-download__warning">
+            This file may not be suitable for users of assistive techonology. <br>
```
Comment (catiarodriguescosta) [inline src/wmnds/components/file-download/_file-download.njk:23]:
```suggestion
            This file may not be suitable for users of assistive technology. <br>
```

**Review by catiarodriguescosta — APPROVED** [review]
Seems to be all good. :)

**Review by houbly — APPROVED** [review]
Looks good to me

**Pull request merged**